server.js

```diff
@@ -0,0 +1,52 @@
+// Define an array with the DCI course modules at the top of the file:
+
+const moduleDCI = ['HTML/CSS', 'JavaScript', 'DOM', 'SPA', 'Backend', 'Fullstack', 'Final Project']
+
+const { request, response } = require('express');
```

Comment:
These imports are actually not necessary. But maybe it was imported by your VSCode somehow accidentally :)